sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -124,6 +302,216 @@ template <typename T> struct ConvertToComp<T, sorting_order::descending> {
 };
 } // namespace detail
 
+namespace radix_sorters {
+
+template <typename ValT, sorting_order OrderT = sorting_order::ascending,
+          unsigned int BitsPerPass = 4>
```

Comment:
Could you please align names with the Spec document? It's important since it's a public API: 
```cpp
    template<typename T,
             sorting_order Order = sorting_order::ascending,
             unsigned int BitsPerPass = 4>
```

Reply:
Fixed